README.md

```diff
@@ -18,72 +18,45 @@
 ```
 
 ## About
-
-Counterfit is a command-line tool and generic automation layer for assessing the security of machine learning systems.
+Counterfit is a generic automation layer for assessing the security of machine learning systems. It brings several existing adversarial frameworks under one tool, or allows users to create their own. 
 
 ### Requirements
-
-- Python 3.7 or 3.8
+- Ubuntu 18.04+
+- Python 3.8
+- Windows is supported by Counterfit, but not necessarily officially supported by each individual framework. 
 - On Windows the [Visual C++ 2019 redistributable](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads) is required
 
-## Getting Started
-
-Choose one of these methods to get started quickly:
-
-- [Option 1: Deploy via Azure Shell](#Option-1-Deploy-now-on-Azure-via-Azure-Shell)
-- [Option 2: Setup an Anaconda Python environment and install locally](#Option-2-Setup-an-Anaconda-Python-environment-and-install-locally)
-
-For more information including alternative installation instructions, please visit our [wiki](https://github.com/Azure/counterfit/wiki).
-
-### Option 1: Deploy via Azure Shell
```

Comment:
Would there be any reason we removed this option? I think this is one of the most popular options to get a taste of Counterfit. IN addition, it's always good to provide as many options as possible for onboarding.

Reply:
@rdheekonda, great point. There were some discrepancies with the internal version and public that caused the info to be missed along the way. I updated the README.md to include this info again 👍 . See 6eb42b59b4f75bc198c6db0e05c9ea663cce4b77